src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests.csproj

```diff
@@ -44,5 +50,8 @@
       <Name>RemoteExecutorConsoleApp</Name>
     </ProjectReference>
   </ItemGroup>
+  <Target Condition="'$(TargetsWindows)' == 'true'" Name="BuildNetfxRemoteInvoker" AfterTargets="BeforeBuild">
+    <MSBuild Projects="RemoteInvokeNetfx\RemoteInvokeNetfx.csproj" Properties="Configuration=Debug; Platform=$(Platform); OutputPath=$(OutputPath); IntermediateOutputPath=$(IntermediateOutputPath)" />
```

Comment:
This is used because we need to build netfx project, if using ProjectReference it would try to build the same configuration or use our build scripts if you override that - in order to use our scripts you would need to perform full netfx build which is not needed in this case